lib/vdc/BOVCollection.cpp

```diff
@@ -7,11 +8,16 @@
 #include <fstream>
 #include <type_traits>
 #include "vapor/VAssert.h"
+#include "vapor/utils.h"
+#include "vapor/FileUtils.h"
 #include <stdio.h>
 
-#ifdef _WINDOWS
+#ifdef WIN32
+    #include <Windows.h>
     #define _USE_MATH_DEFINES
     #pragma warning(disable : 4251 4100)
+#else
+    #include <climits>
 #endif
```

Comment:
Header `climits` seems to be a C++ standard one. Is it not available on Windows so you have to do this complicated trick?

Reply:
Let me check.

Reply:
@shaomeng  It seems this is ok after building and testing on an installer.  File updated.

Note - the windows installer takes > 1 hour to build on cisl-vaporwin10 right now.  Changes like this are hard to test.  God help me if this introduces a regression.